Examples/jbang-catalog.json

```diff
@@ -0,0 +1,50 @@
+{
+  "catalogs": {
+    "example": {
+      "catalog-ref": "C:\\Users\\santo\\Git-projects\\core-edge-chains\\EdgeChains\\Examples\\jbang-catalog.json"
+    }
+  },
+  "aliases": {
+    "wikis": {
+      "script-ref": "wiki\\WikiExample.java",
+      "java-agents": []
+    },
+    "airtable": {
+      "script-ref": "airtable\\AirtableExample.java",
+      "java-agents": []
```

Comment:
Line number 9 and 13 and referring to the java files locally. This is only for demonstration purpose. I will add the actual github urls similar to others when I attach a video indicating that the aliases are working correctly locally too